.github/workflows/tools/apply-patches

```diff
@@ -28,6 +29,10 @@ do
     # applied in reverse
     if git apply -R --check "$patchfile"
     then
+        if [ "x$event" = "xpull_request" ]
```

Comment:
old habits die hard? AFAIK there is no need for `x` prefix if using `""` i.e. 

```suggestion
        if [ "$event" = "pull_request" ]
```

is as good and more straightforward, or am I missing some remaining need for the x?

Reply:
It turns out the `x` is unneeded due to the quotes, so I'll remove it.